index.js

```diff
@@ -3,51 +3,19 @@ const github = require("@actions/github");
 const { execSync } = require("child_process");
 const readFileSync = require("fs").readFileSync;
 
-const { debug, formatLineNumbers } = require("./index.utils");
-const excludedFiles = ["go.mod", "go.sum", "vendor/"];
-
-class Variables {
-  static _instance = null;
-  _variables = {};
-
-  constructor() {
-    if (Variables._instance) {
-      return Variables._instance;
-    }
-
-    const token = core.getInput("github-token", { required: true });
-
-    const quietInput = core.getInput("quiet", { required: false }) || "false";
-    const quiet = ["true", "yes", "on"].includes(quietInput.toLowerCase());
-
-    const pullRequest = github.context.payload.pull_request;
-
-    this._variables = {
-      mainBranch: core.getInput("main-branch", { required: false }) || "main",
-      octokit: github.getOctokit(token),
-      pullRequestAuthor: pullRequest.user.login,
-      pullRequestBranch: pullRequest.head.ref,
-      pullRequestNumber: pullRequest.number,
-      quiet,
-      repo: github.context.repo,
-      token,
-    };
-
-    Variables._instance = this;
-  }
-
-  set(name, value) {
-    this._variables[name] = value;
-  }
-
-  get(name) {
-    return this._variables[name];
-  }
-}
+const { debug } = require("./index.utils");
+const {
+  Variables,
+  getOpenPullRequests,
+  getChangedFiles,
+  requestReviews,
+  requestReviewsInConflictingPRs,
+  createConflictComment,
+} = require("./index.requests");
 
 async function main() {
   if (github.context.payload.pull_request.draft) {
-    debug(`Not running any checks because this pull request is a draft.`)
+    debug(`Not running any checks because this pull request is a draft`)
```

Comment:
Why remove the trailing period?

Reply:
This is to match the rest of the debug() strings, which do not end in periods.